src/main/scala/org/opennetworkinsight/dns/DNSWordCreation.scala

```diff
@@ -1,10 +1,100 @@
 package org.opennetworkinsight.dns
 
+<<<<<<< HEAD
```

Comment:
these are not in the version I just pushed, I will see if I can force the file to be updated